src/seedu/addressbook/commands/Command.java

```diff
@@ -37,6 +37,15 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
     }
 
     /**
+     * Checks if the command have mutated data
+     * @return true for command types that mutate data
+     */
+
+    public boolean isMutating() {
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.

Good job adding comment for `isMutating()`. Since the other sub classes overrides they would need to know how this method functions.